src/supremm/supremmconf.py

```diff
@@ -1,29 +1,41 @@
 #!/usr/bin/env python
 """ helper utiility to print out config info """
+from __future__ import print_function
 
 import sys
 import os
 import json
+import logging
 from getopt import getopt
 
 from supremm.config import Config
+from supremm.scripthelpers import setuplogger
 
 def usage():
     """ print usage """
-    print "usage: {0} [OPTS]".format(os.path.basename(__file__))
-    print "  -s --section SECTION  output the configuration data from the specified section"
-    print "  -i --item ITEM        output the configuration data for the specified item"
-    print "  -h --help             print this help message"
-
+    print("usage: {0} [OPTS]".format(os.path.basename(__file__)))
+    print("  -d --debug            set log level to debug")
+    print("  -c --config           specify the configuration file")
+    print("  -r --resources        output the configuration data for all enabled resources") 
+    print("  -s --section SECTION  output the configuration data from the specified section")
+    print("  -i --item ITEM        output the configuration data for the specified item")
+    print("  -h --help             print this help message")
```

Comment:
Also `--help` is not a valid option. The current code only lets you use `-h`. Suggest that you add the `--help` as a valid alias of `-h` (even though it was not in the scope of the original set of changes.)

Reply:
Added alias for --help.